drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -102,6 +112,19 @@
     top: calc(50% - 16px);
 }
 
+.xblock--drag-and-drop .drag-container .option .numerical-input .spinner-wrapper {
+    position: absolute;
+    right: 0;
+    top: 0;
+    margin-right: 5px;
+    padding-top: 6px;
```

Comment:
@e-kolpakov Any specific advantage(s) of using `padding-top` instead of `margin-top` here? (Just asking because I'm curious :))